<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>471</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>FÃ¼r welche P2P-Plattformen sollen Ergebnisse geladen werden?</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>361</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_bondora">
           <property name="text">
            <string>Bondora</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_grupeer">
           <property name="text">
            <string>Grupeer</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_dofinance">
           <property name="text">
            <string>DoFinance</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBox_iuvo">
           <property name="text">
            <string>Iuvo</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox_peerberry">
           <property name="text">
            <string>PeerBerry</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox_mintos">
           <property name="text">
            <string>Mintos</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="checkBox_robocash">
           <property name="text">
            <string>Robocash</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_estateguru">
           <property name="text">
            <string>Estateguru</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="checkBox_swaper">
           <property name="text">
            <string>Swaper</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="checkBox_twino">
           <property name="text">
            <string>Twino</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>451</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Startdatum</string>
           </property>
           <widget class="QComboBox" name="comboBox_start_month">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>83</width>
              <height>32</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>Jan</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Feb</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mrz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Apr</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mai</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jun</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jul</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Aug</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sep</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Okt</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nov</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dez</string>
             </property>
            </item>
           </widget>
           <widget class="QComboBox" name="comboBox_start_year">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>30</y>
              <width>83</width>
              <height>32</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>2010</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2011</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2012</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2013</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2014</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2015</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2016</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2017</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2018</string>
             </property>
            </item>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Enddatum</string>
           </property>
           <widget class="QComboBox" name="comboBox_end_month">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>83</width>
              <height>32</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>Jan</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Feb</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mrz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Apr</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mai</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jun</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jul</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Aug</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sep</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Okt</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nov</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dez</string>
             </property>
            </item>
           </widget>
           <widget class="QComboBox" name="comboBox_end_year">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>30</y>
              <width>83</width>
              <height>32</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>2010</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2011</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2012</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2013</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2014</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2015</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2016</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2017</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2018</string>
             </property>
            </item>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_start">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>310</y>
          <width>141</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Starte Auswertung</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
